fetch/__init__.py

```diff
@@ -32,25 +33,31 @@ def get_records():
         except Exception as e:
             print(e)
 
-        try:
-            print(f"Crawling {eu.SOURCE}...")
-            data.update(eu.find(query))
-        except Exception as e:
-            print(e)
+        #try:
+        #    print(f"Crawling {eu.SOURCE}...")
+        #    data.update(eu.find(query))
+        #except Exception as e:
+        #    print(e)
 
-        try:
-            print(f"Crawling {isrctn.SOURCE}...")
-            data.update(isrctn.find(query))
-        except Exception as e:
-            print(e)
+        #try:
+        #    print(f"Crawling {isrctn.SOURCE}...")
+        #    data.update(isrctn.find(query))
+        #except Exception as e:
```

Comment:
not merge-able if all of this stuff is commented out :)

Reply:
Will fix before final commit